### DISCUSSION TITLE
Social links - Add support for Flickr

### DISCUSSION THREAD

**Discussion by heitorPB** [community/community#54746] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Currently, there are many external social links that shows the icons and a "pretty" link to the user profile. But [Flickr](https://flickr.com) is not supported :( I'd like to request support for Flickr.

Here's how it currently shows on my profile:

![flickr](https://user-images.githubusercontent.com/13461702/236702644-d11f167e-8c10-4e30-9145-a40416f2fc0b.png)

The Flickr Brand Guidelines are available [here](https://www.flickrhelp.com/hc/en-us/articles/4404071066260-Brand-guidelines).

**Comment by pjpscriv** [comment]
Hey @heitorPB I created a chrome extension as a workaround for this - [GitHub Social Icons](https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp)

Flickr looks like this ☺️

![image](https://github.com/user-attachments/assets/fda91b15-a429-4e72-9681-3253efa96580)
